### PR TITLE
Recipe generation 

### PR DESCRIPTION
**Pull request by blaz2f**
Recipe generation fuction using input user from POST request and database table ingredients

### CODE REVIEW

File: model/createRecipeTestSample.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+{
+ "user_id":15,
+ "recipe_name":"Tomato pesto chicken pasta",
+ "cuisine":"Italian",
```
Comment (sicsixdkn) [inline model/createRecipeTestSample.json:5]: I think cuisine should probably be an id as well

**Review by sicsixdkn — CHANGES_REQUESTED** [review]
Looks good. I think cuisine will probably need to be an ID as well though

File: model/createRecipeTestSample.json (hunk 1 above)
Comment (blaz2f) [reply model/createRecipeTestSample.json]: done!

**Review by sicsixdkn — APPROVED** [review]
Looks good!

**Pull request merged**